sale_channel_hook/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
sale channel hook should not be here move it in hook PR

Reply:
fixed